examples/pytorch/BertNewsClassification/README.md

```diff
@@ -0,0 +1,75 @@
+## Bert Sentiment Analysis example with MLFlow
+
+In this example, we have used a pretrained Bert model to classify the news reports into any one of the 4 different categories("world", "Sports", "Business", "Sci/Tech").
+The autolog code uses Pytorch Lightning's MLFlowLogger to log metrics.
+The code is trained using pytorch lightning loop and the autolog function call in the main - `autolog()`
+is responsible for logging the params, metrics, model summary and the model.
+ 
+
+### Code related to MLflow:
+* [`mlflow.pytorch.pytorch_autolog`]
```

Comment:
```suggestion
* [`mlflow.pytorch.pl_autolog`]
```

Are we sure about `mlflow.pytorch.pytorch_autolog` as the API name? If we were to follow the pattern for other frameworks, we'd do `mlflow.pytorch_lightning.autolog`

Reply:
`mlflow.pytorch_lightning.autolog` looks better. I will rename it :-)

Reply:
@shrinath-suresh thanks! Actually sorry for the flip-flop on this - since Pytorch lightning autologging does in fact call `mlflow.pytorch.log_model` (and therefore generates the `pytorch` flavor / saves models that can be loaded back via `mlflow.pytorch.load_model`), we're thinking we should actually just call the API `mlflow.pytorch.autolog`, and mention in the docstring that only pytorch-lightning is supported for now

Reply:
Then the user workflow would be:

```
import mlflow.pytorch
mlflow.pytorch.autolog()

# Define model using pytorch lightning
class LitAutoEncoder(pl.LightningModule):
   ...

model = LitAutoEncoder(...)
# Fit model etc
...

# Later, load back model using mlflow.pytorch.load_model
autologged_pl_model = mlflow.pytorch.load_mode("/path/to/logged/model")
```

Reply:
@smurching Should we log a model as a scripted model so we can serve it using the torchserve plugin?

https://pytorch-lightning.readthedocs.io/en/latest/production_inference.html#exporting-to-torchscript

Reply:
@harupy Scripting is optional path and not all models will be torchscripted. We support both eager mode and torchscripted mode so it will be upto the user which option then want to go with based on their specific use case and needs. TorchScripting is recommended for deploying in production, however not all models are scriptable and some hardwares require conversion of models to ONNX/TVM etc so we should not be forcing users down a path.

Reply:
> @shrinath-suresh thanks! Actually sorry for the flip-flop on this - since Pytorch lightning autologging does in fact call `mlflow.pytorch.log_model` (and therefore generates the `pytorch` flavor / saves models that can be loaded back via `mlflow.pytorch.load_model`), we're thinking we should actually just call the API `mlflow.pytorch.autolog`, and mention in the docstring that only pytorch-lightning is supported for now

@smurching @harupy 
Tried moving autolog method to root directory as `pytorch.py`. However, we already have a folder named `pytorch` under root directory and all our log/save model codes are residing in it. When i try to do `mlflow.pytorch.autolog()` it looks into `pytorch` folder and fails. 

In case of keras and fastai, the folders are not present. The entire autolog code is placed as `keras.py` and `fastai.py` so that it can be directly imported as `mlflow.keras.autolog`. 

Technically, the import should be mlflow.pytorch.<FILENAME>.autolog. Any thoughts ?

Reply:
@shrinath-suresh good question, could we do something similar to what we do for Spark autologging ([link](https://github.com/mlflow/mlflow/blob/64c93c81167b8a132bf00535fc23f3420c9caf7f/mlflow/spark.py#L630-L688))? Namely, define the actual autologging implementation in an internal `_autolog` method under a module like `mlflow.pytorch._pytorch_autolog`, then import and expose it as `def autolog` in `mlflow/pytorch/__init__.py`?

Reply:
@smurching  Added the wrapper method in `__init__.py` and i am able to use it as `mlflow.pytorch.autolog()`. Thanks for the suggestion and link.